### DISCUSSION TITLE
Curator Presets

### DISCUSSION THREAD

**Discussion by fabianpimminger** [awcodes/filament-tiptap-editor#368] [Q&A]
Hi,

Is there a way to insert specific presets from curator in tiptap?

For example a user changes the focal point or crop of an image and wants to insert this variant into the editor?

**Comment by awcodes** [comment]
Currently there is not. The curations aren't available via the picker / modal.

**Reply by fabianpimminger** [reply]
Ok :( … that would be a great addition for a future release.
